llama.cpp

```diff
@@ -4319,10 +4424,8 @@ static bool llm_load_tensors(
                         layer.attn_norm   = ml.create_tensor(ctx_layer, tn(LLM_TENSOR_ATTN_NORM, "weight", i), {n_embd});
                         layer.attn_norm_b = ml.create_tensor(ctx_layer, tn(LLM_TENSOR_ATTN_NORM, "bias", i),   {n_embd});
 
-                        if (gguf_find_tensor(ml.ctx_gguf, tn(LLM_TENSOR_ATTN_NORM_2, "weight", i).c_str()) >= 0) {
```

Comment:
I'm not sure if removing this will affect other models (non-llama?)

Reply:
It is a good change, uses the `required` parameter instead of checking. Maybe it should also have a check to ensure that either none of the two tensors exist, or both do, but not only one.